pkg/actor/actor.go

```diff
@@ -56,7 +56,7 @@ type Mailbox interface {
 
 	// Try to receive a message.
 	// It must be nonblocking and should only be called by System.
-	tryReceive() (message.Message, bool)
+	Receive() (message.Message, bool)
```

Comment:
Base on the comment, TryReceive is better?

Reply:
I'll update comment, as we have `Send` for nonblocking send, we should name `Receive` for nonblocking receive.